MultiPageImporter.jsx

```diff
@@ -484,11 +486,11 @@ else if(reverseOrder && noPDFError)
 }
 else
 {
-	addPages(docStartPG, startPG, endPG);
+	addPages(docStartPG, startPG, endPG, skipVar);
 }
 
 // Kill the Object style
-tempObjStyle.remove();
+//tempObjStyle.remove();
```

Comment:
I wonder why you removed this one? 
I don't really see the connection with the feature you implemented.